lib/containers/k8s.pm

```diff
@@ -49,6 +49,7 @@ sub uninstall_k3s {
 sub install_kubectl {
     if (is_sle) {
```

Comment:
Instead of this, I would add a check if `kubectl` command exists, and exit if so. 
BCI tests on K8s are running on PC tools image, with kubectl pre-installed, so you don't need to install it again, so you can skip all this module enablement.

Reply:
mmm but right now the kubectl installed on pc-tools is container based, but is not working yet